content/en/docs/v3.5/dev-guide/grpc_naming.md

```diff
@@ -12,15 +12,23 @@ The etcd client provides a gRPC resolver for resolving gRPC endpoints with an et
 
 ```go
 import (
-	clientv3 "go.etcd.io/etcd/client/v3"
```

Comment:
This may be nice to keep because the module is named "clientv3" and not "v3"; therefore, `goimports` sets it as a named import with "clientv3". So, the suggestion is to keep this line and also for v3.6.

Reply:
got it, revert it to fixed, thank you @ivanvc!